Gemfile

```diff
@@ -102,7 +102,8 @@ gem("unicorn")
 # Use i18n for internationalization
 gem("i18n")
 
-# Enable Textile markup language. See http://redcloth.org/
+# Enable Textile markup language. See https://github.com/jgarber/redcloth,
```

Comment:
Redcloth org is an expired domain. http://redcloth.org/
I wonder if the RedCloth gem is compatible with changes to the Textile standard
(the last change to the gem was over 5 years ago), 
and what might happen going forward.